### PR TITLE
Update docs for PropertyManager

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/System.Windows.Forms/PropertyManager.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>When overridden in a derived class, adds a new item to the underlying list.</summary>
+        <summary>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/PropertyManager.xml]: Should this have an \<exception> item?

File: xml/System.Windows.Forms/PropertyManager.xml [hunk 2]
```diff
@@ -335,7 +335,7 @@
       </Parameters>
       <Docs>
         <param name="index">The index of the row to delete.</param>
-        <summary>When overridden in a derived class, deletes the row at the specified index from the underlying list.</summary>
+        <summary>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</summary>
```
Comment (rpetrusha) [inline xml/System.Windows.Forms/PropertyManager.xml]: Should this have an \<exception> item?

**Comment by mairaw** [comment]
@merriemcgaw @Tanya-Solyanik do you want to take a look at the changes I made for PropertyManager?

**Comment by mairaw** [comment]
Correct @Tanya-Solyanik. This file was part of the changes in PR #2974 but I thought the particular change for this class was not relevant (https://github.com/RussKie/dotnet-api-docs/commit/cd758031dd1ea52938230e89e70c2771dc10042d#diff-1283fb4f83336c047d77c7321c54368c). But I decided to review the class anyway to make sure it was accurate and following style guide rules.

**Pull request merged**